CloudCannon/bookshop
# Adding support for `npx @bookshop/cli --init` command

Testing:

* Clone and checkout
* run `npm link` inside `bookshop/cli` folder
* Go to another empty folder
* Run `npx @bookshop/cli --init .`

Example storybook (current master) should appear.

*Once PR is merged*, the example will no longer feature `file:../../` links in package.json
